tests/test_cli.py

```diff
@@ -137,7 +137,13 @@ def test_validate_default(self) -> None:
     def test_validate_default_with_require_description(self) -> None:
         """Test validate command with description required."""
         runner = CliRunner()
-        result = runner.invoke(cli, ["validate", "--require-description", TEST_YAML], catch_exceptions=False)
+        try:
+            # Wrap this in a try/catch in case an exception is thrown.
+            result = runner.invoke(
+                cli, ["validate", "--require-description", TEST_YAML], catch_exceptions=False
+            )
+        except Exception as e:
+            print(e)
```

Comment:
Hmm, you probably want to rethrow the exception here, so that test fails. ` Schema.require_description(False)` should go into `finally` then.

Reply:
Good catch. I've made the change as you suggested.